src/azure-cli-core/tests/test_profile.py

```diff
@@ -708,8 +736,8 @@ def get_auth_context(authority, **kwargs):  # pylint: disable=unused-argument
 
         # action
         mgmt_resource = 'https://management.core.windows.net/'
-        token_type, token = creds_cache.retrieve_token_for_user(self.user1, self.tenant_id,
-                                                                mgmt_resource)
+        token_type, token, entry = creds_cache.retrieve_token_for_user(self.user1, self.tenant_id,
```

Comment:
nit: technically `entry` -> `_` as it's not used right? I guess pylint doesn't run on test files.